web/app/models/ability.rb

```diff
@@ -0,0 +1,32 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the passed in user here. For example:
+    #
+      user ||= User.new # guest user (not logged in)
+      if user.role == 'admin';
+        can :manage, :all
+      else
+        can :read, :all
+      end
+    #
+    # The first argument to `can` is the action you are giving the user 
```

Comment:
this file was generated with `rails generate CanCan:Ability`, and i just didn't bother to modify it.  i'm impartial to comments, we can just delete them and use comments here to communicate :)